ui/app/components/app-card/release.hbs

```diff
@@ -4,19 +4,6 @@
       <p>
         <b class="badge badge--version">v{{@model.sequence}}</b>
         <code>{{truncate @model.id 8 false}}</code>
-        {{#if (eq @model.status.state 1)}}
```

Comment:
if (hypothetically) a plugin does not not provide status reports, would we want to keep the `state` badges? (cc @briancain)

Reply:
If I remember right, we decided to hide the badge if no status report was returned from the plugin. Otherwise we don't really have a "state" to show that would imply it's missing or unimplemented.

Reply:
This is addressed by WP-071, which will make the distinction between status and status report clearer.

Reply:
More on that: I’m hoping the WP-071 work will follow shortly after this, so we can consider this a stopgap solution.